zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassieSpanStoreFactory.scala

```diff
@@ -119,12 +119,16 @@ trait CassieSpanStoreFactory { self: App =>
   val cassieMaxTraceCols = flag("zipkin.store.cassie.maxTraceCols", Defaults.MaxTraceCols, "max number of spans to return from a query")
   val cassieReadBatchSize = flag("zipkin.store.cassie.readBatchSize", Defaults.ReadBatchSize, "max number of rows per query")
 
+  val cassieUsername = flag("zipkin.store.cassie.username", "", "cassandra username")
```

Comment:
i think these user/pass are per-keyspace not per-cassandra, seems weird to call this "cassandra username".